threem.py

```diff
@@ -70,7 +71,10 @@ def get_events_between(self, start, end, cache_result=False):
         try:
             events = EventParser().process_all(response.content)
         except Exception, e:
-            print response.content
+            self.log.error(
```

Comment:
Okay, so this self.log needs to be changed to `logging.error`. All the others are good to go.